stories/components/person/person.properties.stories.js

```diff
@@ -313,6 +313,24 @@ export const personDisableImageFetch = () => html`
    <mgt-person person-query="me" disable-image-fetch></mgt-person>
  `;
 
+export const personDisablePresenceRefresh = () => html`
```

Comment:
This modern Christmas tree is amazing! I love the approach, it's perfect!

Only thing I'd add, is that in the latest version of our Storybook we support signing in, so I'd love to see one "without" the mocked refresh, just so I could open up teams and change my status and I'd see the results.

Reply:
I didn't see any of the person stories that were using the signed-in component. Can you point me to an example of what you were looking for?

Reply:
Just use the "Sign in" button on top of Storybook, you should be good to go!

![image](https://github.com/microsoftgraph/microsoft-graph-toolkit/assets/7620955/753e633f-8d53-4ea3-ba9a-9f98023c1584)

Reply:
I changed (mock) to (fake) and I created a "My Presence" page under Properties with the mgt-login component. However, it is still Megan Bowen, not Peter Lasne, so that is being mocked somehow.